Apps/Admin/Client/Pages/Covid19TreatmentAssessmentPage.razor

```diff
@@ -13,7 +13,7 @@
 
 <MudLink Href="@PatientDetailsUrl">
     <MudText Typo="Typo.h5" class="mb-4">
-        <MudIcon Icon="fas fa-arrow-left" /><span class="ml-2">Back</span>
+        <MudIcon Icon="fas fa-arrow-left" data-testid="back-icon" /><span class="ml-2">Back</span>
```

Comment:
I'd suggest moving the data-testid to the MudLink element and renaming it `back-button`.

Reply:
Changed.